src/main/java/frc/robot/superstructure/Tolerances.java

```diff
@@ -9,4 +9,6 @@ public class Tolerances {
 	public static final double TRANSLATION_TOLERANCE_METERS = 0.05;
 	public static final double TRANSLATION_VELOCITY_TOLERANCE = 0.05;
 
+	public static final Rotation2d PIVOT_POSITION_TOLERANCE = Rotation2d.fromDegrees(5);
```

Comment:
use dont use it so delete

Reply:
will use in superstructure, toav said it so im resolving